core/src/main/java/nl/nn/adapterframework/jta/narayana/NarayanaDataSourceFactory.java

```diff
@@ -29,21 +29,31 @@
 import com.arjuna.ats.internal.jdbc.drivers.modifiers.ModifierFactory;
 import com.arjuna.ats.jta.recovery.XAResourceRecoveryHelper;
 
+import lombok.Getter;
 import lombok.Setter;
 import nl.nn.adapterframework.jndi.JndiDataSourceFactory;
+import nl.nn.adapterframework.util.AppConstants;
 import nl.nn.adapterframework.util.LogUtil;
 
 public class NarayanaDataSourceFactory extends JndiDataSourceFactory {
 	protected static Logger log = LogUtil.getLogger(NarayanaDataSourceFactory.class);
 
+	private @Getter @Setter int maxPoolSize=20;
+
 	private @Setter NarayanaJtaTransactionManager transactionManager;
 
+	public NarayanaDataSourceFactory() {
```

Comment:
Is it possible to override `toString()` for this class as well and print out the actual pool size? Or is that not supported by Narayana?